spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/otlp/OtlpProperties.java

```diff
@@ -70,6 +75,14 @@ public void setTimeout(Duration timeout) {
 		this.timeout = timeout;
 	}
 
+	public Transport getExporter() {
```

Comment:
The field is named `transport` and the enum, too. Can we name that getter `getTransport`, too? This would change the property to `management.otlp.tracing.transport=grpc`.

Reply:
Fixed the getter/setter.